### PR TITLE
Add stubs for new services

### PR DESCRIPTION
**Pull request by gtema**
A plain stubs (initial service structure, empty proxies, initial tests, func tests for the proxy)
Currently plas, dds, lts blacklisted, since they are missing in the service catalog.

### CODE REVIEW

File: .stestr.blacklist.functional [hunk 1]
```diff
@@ -6,3 +6,6 @@ otcextensions.tests.functional.sdk.kms*
 otcextensions.tests.functional.sdk.volume_backup*
 otcextensions.tests.functional.sdk.volume_backup*
```
Comment (vineet-pruthi) [inline .stestr.blacklist.functional]: added twice

File: otcextensions/sdk/__init__.py [hunk 2]
```diff
@@ -98,17 +122,28 @@
...
     'volume_backup': {
         'service_type': 'volume_backup',
         'append_project_id': True,
         'endpoint_service_type': 'vbs',
     },
+    'waf': {
+        'service_type': 'waf',
+        'append_project_id': True,
+    }
 }
```
Comment (vineet-pruthi) [inline otcextensions/sdk/__init__.py:151]: Missing rts, evs, elb, workspace, vpcep and sdrs

Comment (gtema) [reply otcextensions/sdk/__init__.py:151]: rts - upstream orchestrate, evs - upstream block_storage, elb - upstream "neutron LB" hybrid, workspace - likely soon deprecated
what is vpcep?

File: .stestr.blacklist.functional (hunk 1 above)
Comment (gtema) [reply .stestr.blacklist.functional]: done

**Comment by gtema** [comment]
recheck

File: otcextensions/sdk/__init__.py (hunk 2 above)
Comment (gtema) [reply otcextensions/sdk/__init__.py:151]: sdrs added, vpcep is still missing in the docs. I don't think it make sense to add stubs for services which might/might not be deployed to the platform in months/years. Let's have proxies for everything, what we have now.

Comment (vineet-pruthi) [reply otcextensions/sdk/__init__.py:151]: Agree, we can wait until the VPCEP is released

File: otcextensions/sdk/smn/smn_service.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+from openstack import service_description
+
+from otcextensions.sdk.smn.v1 import _proxy
+
+
+class SmnService(service_description.ServiceDescription):
+    """The SMN service."""
+
+    supported_versions = {
+        '1': _proxy.Proxy
```
Comment (vineet-pruthi) [inline otcextensions/sdk/smn/smn_service.py]: smn api is v2

Comment (gtema) [reply otcextensions/sdk/smn/smn_service.py]: done

**Review by vineet-pruthi — DISMISSED** [review]

**Review by tischrei — DISMISSED** [review]
I can't identify any issues, but my knowledge is also limited

**Pull request merged**